client/client.go

```diff
@@ -34,18 +34,16 @@ type responseBody struct {
 	} `json:"errors"`
 }
 
-type Invoker interface {
-	Invoke(context.Context, *lambda.InvokeInput, ...func(*lambda.Options)) (*lambda.InvokeOutput, error)
-}
-
-type Client struct {
-	invoker Invoker
+type LambdaClient struct {
+	invoker interface {
+		Invoke(context.Context, *lambda.InvokeInput, ...func(*lambda.Options)) (*lambda.InvokeOutput, error)
+	}
```

Comment:
I'm having a hard time understanding the benefit of this over the separate `Invoker` interface `Client` struct. What's the advantage of this pattern?

Reply:
Gotcha, I think in-lining it is a little non standard so maybe the `GqlClient` approach would be better?

Reply:
Sorry it's been a little crazy but I finally got around to it 😁